src/components/media-player.js

```diff
@@ -99,6 +141,8 @@ function MediaPlayer({
 
```

Comment:
Seems `trackRef.current` is not initialized on the initial setting of the video source and it throws on line 138:

<img width="658" alt="Screen Shot 2022-07-20 at 09 44 36" src="https://user-images.githubusercontent.com/8189923/179926018-7b84a1b3-e769-43cb-b8ab-0d1f7b42da08.png">

I can reproduce this only when setting the video for the first time, on the newly created `interactive-media` section.

Reply:
Very good catch, fixed it. Thanks!